rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/Diagnostics.java

```diff
@@ -104,7 +104,7 @@ public static List<Template> generateParseErrorDiagnostics(ITree errorTree) {
         List<Template> diagnostics = new ArrayList<>();
 
         // Highlight selected parts of the error tree
-        if (ERROR_LOCATION_HIGHLIGHT != null) {
+        if (ERROR_LOCATION_HIGHLIGHT != null) { // NOSONAR
```

Comment:
I tried to squeeze in `// NOSONAR` comments before but the general consensus was that it is not a good idea to introduce tool-specific comments.

Reply:
and more over, I agree with sonar, this is dead code, we should not forget this.